### PR TITLE
React Testing Library Configuration

### PR DESCRIPTION
**Pull request by LiorKGOW**
Followed [this guide](https://testing-library.com/docs/react-testing-library/setup) (without using Jest) 
and [these installation instructions](https://github.com/testing-library/react-testing-library#installation)

Todo List:
- [x] Add configuration
- [x] Add example tests
- [ ] Add Github Action

### CODE REVIEW

**Review by Ron-Lavi — COMMENTED** [review]
Can you add an example test and GitHub action?
we can merge first #8 so you can already create the test for that

**Comment by LiorKGOW** [comment]
> Can you add an example test and GitHub action? we can merge first #8 so you can already create the test for that

I have added some example tests, you can inspect them in the second commit 👍🏼 

Do you think I need this?: https://github.com/kentcdodds/jest-cypress-react-babel-webpack/issues/20#issuecomment-504697112 @Ron-Lavi

**Comment by LiorKGOW** [comment]
@Ron-Lavi 
I came across this guide: https://dev.to/achukka/build-and-test-react-app-with-github-actions-2jb2 to setup a Github Action to test React with react testing library, but according to the checks I added, there were problems in node_modules & package-lock.json files, which were in gitignore
It found a syntax error in one of the files in node_modules, which is not on Github
Should I add it just for this fix and remove it in another PR?

File: .github/workflows/build_test_react.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm ci, build and test
```
[the remark this answers is not in the document]
Comment (Ron-Lavi) [reply .github/workflows/build_test_react.yml:29]: it should be something like:
- yarn
- yarn test

File: .github/workflows/build_test_react.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: yarn ci, build and test
+        run: |
+          yarn ci
```
Comment (Ron-Lavi) [inline .github/workflows/build_test_react.yml:31]: Just `yarn`

File: .github/workflows/build_test_react.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          yarn run build --if-present
```
[the remark this answers is not in the document]
Comment (LiorKGOW) [reply .github/workflows/build_test_react.yml:32]: Sure

File: .github/workflows/build_test_react.yml (hunk 2 above)
Comment (LiorKGOW) [reply .github/workflows/build_test_react.yml]: Fixed

File: .github/workflows/build_test_react.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (LiorKGOW) [reply .github/workflows/build_test_react.yml]: build Removed 👍🏼

**Pull request merged**